recipes/default.rb

```diff
@@ -60,6 +60,11 @@
   only_if { platform?('ubuntu') }
 end
 
+file '/etc/fail2ban/jail.d/00-firewalld.conf' do
+  action 'delete'
+  only_if { platform?('centos') }
```

Comment:
Should this happen on all the RHEL like platforms?

Reply:
I'm not sure...  I've only tested against or seen this file on CentOS 7... Above only checks for Ubuntu and not all Debian based machines.